mozilla-ai/lumigator
# Changed ordering of commands in make local-up

## What's changing

`Makefile` was running docker compose as an attached process, so the `uv run pre-commit install` command was not executed until docker compose was stopped.
The change swaps the two commands so `uv` always runs before the attached docker compose.

## How to test it
`make local-up` should show `uv run pre-commit install` as the first executed command

## Additional notes for reviewers

## I already...

- [ ] added some tests for any new functionality (not needed)
- [ ] updated the documentation (not needed)
- [ ] checked if a (backend) DB migration step was required and included it if required (not needed)
